backend/init_db.py

```
@@ -31,184 +31,98 @@ def migrate_db():
     """
     try:
         with engine.connect() as conn:
+            def safe_migrate(sql, success_msg):
+                try:
+                    conn.execute(text(sql))
+                    conn.commit()
+                    if success_msg:
+                        logger.info(success_msg)
+                except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)